signac/contrib/migration/v1_to_v2.py

```diff
@@ -79,12 +81,10 @@ def _migrate_v1_to_v2(root_directory):
 
     # Now move all other files.
     files_to_move = {
-        ".signac_shell_history": os.sep.join((".signac", "shell_history")),
-        ".signac_sp_cache.json.gz": os.sep.join(
-            (".signac", "statepoint_cache.json.gz")
-        ),
+        ".signac_shell_history": os.path.join(".signac", "shell_history"),
+        ".signac_sp_cache.json.gz": os.path.join(".signac", "statepoint_cache.json.gz"),
     }
     for src, dst in files_to_move.items():
-        src = os.sep.join((root_directory, src))
+        src = os.path.join(root_directory, src)
```

Comment:
I'm fine making the switch, but to be clear we also don't benefit from the extra checks in this case since we control the inputs.

Reply:
Right. It's just about not using a "performance hack" (which we previously decided needed a code comment for explanation in every instance it was used) where a more common function can do the job correctly.